sdk/js-connect/core/base/src/constants/chains.ts

```diff
@@ -0,0 +1,99 @@
+import { MapLevel, zip } from "../utils";
+import { constMap } from "../utils/mapping";
+
+const chainsAndChainIdEntries = [
+  //Unlike the old sdk, we are not including an "Unset" chain with chainId 0 here because:
+  //  * no other types would be associated with it (such as contracts or a platform)
+  //  * avoids awkward "chain but not 'Unset'" checks
+  //  * "off" is not a TV channel either
+  //Instead we'll use `null` for chain and 0 as the chainId where appropriate (e.g. governance VAAs)
+  ["Solana", 1],
+  ["Ethereum", 2],
+  ["Terra", 3],
+  ["Bsc", 4],
+  ["Polygon", 5],
+  ["Avalanche", 6],
+  ["Oasis", 7],
+  ["Algorand", 8],
+  ["Aurora", 9],
+  ["Fantom", 10],
+  ["Karura", 11],
+  ["Acala", 12],
+  ["Klaytn", 13],
+  ["Celo", 14],
+  ["Near", 15],
+  ["Moonbeam", 16],
+  ["Neon", 17],
+  ["Terra2", 18],
+  ["Injective", 19],
+  ["Osmosis", 20],
+  ["Sui", 21],
+  ["Aptos", 22],
+  ["Arbitrum", 23],
+  ["Optimism", 24],
+  ["Gnosis", 25],
+  ["Pythnet", 26],
+  ["Xpla", 28],
+  ["Btc", 29],
+  ["Base", 30],
+  ["Sei", 32],
+  ["Rootstock", 33],
+  ["Wormchain", 3104],
+  ["Cosmoshub", 4000],
+  ["Evmos", 4001],
+  ["Kujira", 4002],
+  // holy cow, how ugly of a hack is that?! - a chainId that's exclusive to a testnet!
+  ["Sepolia", 10002],
+] as const satisfies MapLevel<string, number>;
```

Comment:
Does this mean we are going to have yet another place to maintain this list?

Thinking outloud, does it make sense to have a clearly namespaced singular wormhole typescript lib that is solely the constants that can be imported by this new sdk and the "legacy" sdk, or is the legacy one going to be fully deprecated?

Reply:
I'm hopeful we end up deriving the "legacy" config/consts from this one so we only have to update it/reference it from one spot. 

If we do have a single Wormhole ts lib, this one is its own module so should be importable to access the config

Reply:
the plan is that this SDK is the one to rule them all. the legacy one in the monorepo, as well as the `sdk` directory inside `wormhole-connect`, will both be replaced by this.